stdlib/printexc.mli

```diff
@@ -157,7 +157,7 @@ external raise_with_backtrace: exn -> raw_backtrace -> 'a
 
 (** {1 Current call stack} *)
 
-val get_callstack: int -> raw_backtrace
+external get_callstack: int -> raw_backtrace = "caml_get_current_callstack"
```

Comment:
I think I understand this change from `val` to `external`, but it would be nice to explain it in this PR so that I don't have to guess your intent.

Reply:
I updated the description of this PR.